Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Web server has a dependency on node
+FROM node:latest as node
```

Comment:
I don't think you need node to run the server. I don't have node installed on my machine at all and it works